APIs/openEO/Glossary.qmd

```diff
@@ -47,10 +53,29 @@ In the EO domain, it is common to be implicit about the temporal dimension and j
 Special cases are raster and [vector datacubes](https://r-spatial.org/r/2022/09/12/vdc.html){target="_blank"}.
 Learn more about datacubes in the [datacube documentation](https://openeo.org/documentation/1.0/datacubes.html){target="_blank"}.
 
+## Vector data
+
+In general, **vector data** represent specific things (also called "features") in a space, e.g. on the surface of the Earth.
```

Comment:
This description also matches raster data

Reply:
updated the sentence